scrunch/datasets.py

```diff
@@ -375,6 +379,121 @@ def get_dataset(self, dataset):
         ds = BaseDataset(shoji_ds)
         return ds
 
+    def create_project(self, name):
+        # This should be a method of the Project class
+        proj_res = self.resource.create(shoji_entity_wrapper({
+            'name': name
+        }))
+        return Project(proj_res)
+
+    # Compatibility method to comply with Group API
+    create_group = create_project
+
+    @property
+    def is_root(self):
+        return bool(getattr(self.resource, 'parent'))
```

Comment:
Where is this `parent` attribute coming from?

Reply:
Fixed! https://github.com/Crunch-io/scrunch/commit/346044a8cc0dcf435925d19ff929025c5efed298